src/index.css

```diff
@@ -1,13 +1,11 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:wght@400;500;600;700&family=DM+Serif+Text:ital,wght@1,400;1,500;1,600&family=Poppins:wght@400;500;600;700&family=Roboto+Condensed:wght@400;500;600;700&family=Roboto:ital,wght@0,400;0,500;0,600;0,700&display=swap');
```

Comment:
What was the issue here? why did you hve remove this?

Reply:
Font is not working properly Its medium light lighter is not working but after the code change this works properly now